### PR TITLE
feat(calls): EN-7737, EN-8123: VOIP call logs

### PR DESCRIPTION
**Pull request by glenwperry**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "syncromatics-track-api",
-  "version": "0.0.0-development",
+  "version": "3.27.1-EN-7737-beta.106",
```
Comment (thzinc) [inline package.json:3]: This should be reverted before merging

File: src/resources/CallLog.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
+import Resource from './Resource';
+
+/**
+ * CallLog resource
+ */
+class CallLog extends Resource {
```
Comment (thzinc) [inline src/resources/CallLog.js:6]: This should be either `VoipCallRecord` (or perhaps `CallRecord`) to be in line with the models used in Track API

**Review by thzinc — CHANGES_REQUESTED** [review]

**Comment by thzinc** [comment]
Resolves issue EN-8123

**Review by BorisGanov-GMV — APPROVED** [review]
LGTM

**Review by thzinc — APPROVED** [review]

**Pull request merged**